clang/docs/checkedc/Update-to-latest-LLVM-sources.md

```diff
@@ -204,6 +213,27 @@ checkedc-specific changes and llvm-11-to-12 changes:
     - The pristine LLVM/Clang source at &lt;branch-point-of-11-on-main&gt;
     - The pristine LLVM/Clang source at &lt;branch-point-of-12-on-main&gt;
 
+    The description of the three snapshots above holds only for the initial 
+    merge from LLVM 11 to 12 in phase 1 step 6. For a merge of additional
+    Checked C feature development in phase 1 step 8, the relevant versions are a
+    snapshot of branch `updated_baseline_master_12` after step 6 or an earlier
+    iteration of step 8 (i.e. the common ancestor), a snapshot of the current
+    `master` branch that is being merged, and a snapshot of the current
+    `updated_baseline_master_12` branch before starting the merge.
```

Comment:
I don't think this description of the relevant versions is correct.  See this diagram of two successive merges:
```
LLVM 11 -> master (step 6)      -> master (step 8)
   |              |                       |
   v              v                       v
LLVM 12 -> ubm12 (after step 6) -> ubm12 (after step 8)
```
IIUC, the versions in the document correspond to "ubm12 (after step 6)", "master (step 8)", and `updated_baseline_master_12` after step 7 (which could differ from its value after step 6 due to manual cherry-picks and fixes).  But from the diagram, it seems clear that the common ancestor is "master (step 6)" and the other two versions are "master (step 8)" and "ubm12 (after step 6)".

Perhaps this illustrates the point that we should use the three versions identified by Git rather than trying to determine them manually and potentially making mistakes.  I realize now that it's possible to get the commit IDs via `$(git merge-base HEAD MERGE_HEAD)`, `HEAD`, and `MERGE_HEAD`.  So we can still "use source snapshots" but query the commit IDs from the in-progress merge rather than determine them manually.

Reply:
Very true! I have incorporated your review comments. Thanks for the review!